.gitignore

```diff
@@ -24,6 +24,7 @@ jujugui.egg-info/
 jujugui/static/build/*
 jujugui/static/gui/build/
 karma-mocha-old.conf.js
+karma.conf.js
```

Comment:
Is this something we generate? Just to make sure, since it wasn't excluded before.

Reply:
before we weren't generating it, it was just there. But now we only want to keep the template around.